cmd/datamon/cmd/config.go

```diff
@@ -19,31 +21,23 @@ type CLIConfig struct {
 	Context    string `json:"context" yaml:"context"`       // Context for datamon
 }
 
-func newConfig() (*CLIConfig, error) {
-	var config CLIConfig
-	err := viper.Unmarshal(&config)
-	if err != nil {
-		return nil, err
-	}
-	return &config, nil
-}
-
-func (c *CLIConfig) setDatamonParams(flags *flagsT) {
-	if flags.context.Descriptor.Name == "" {
-		flags.context.Descriptor.Name = c.Context
-	}
+func populateRemoteConfig() {
+	var flags *flagsT
+	flags = &datamonFlags
```

Comment:
```
flags := &datamonFlags
```

Reply:
this is an ongoing stylistic discussion:  whether to hindley-milner or not to hindley-milner.

from previous conversations with ritesh, i've variously considered type inference super and harmful.

i do not have a thorughgoing, apply-everywhere stance, except i do recall that i wanted explicit declaration by type separate from assignment here for some reason.  i think simple b/c i found it difficult to remember, in whatever context of a mental model i have for the `cmd/` package environment what with the rapid proliferation of global non-function variables, the type assumptions on `datamonFlags` at this particular part of the codebase, and i actually wanted the reminder.

Reply:
ok. Use // nolint hint then